### DISCUSSION TITLE
terraform-module manager using wrong lookup name

### DISCUSSION THREAD

**Discussion by samithShetty** [renovatebot/renovate#10600] [Request Help] [answered]
Hello,

I'm currently using a self-hosted instance of renovate 25.49 in bitbucket-server, and I have an issue using the terraform manager where it detects the modules, but incorrectly looks them them up. Here's a snippet of of the debug logs of one of the terraform module lookups failing (they all follow this format)
```
"packageFile": "main.tf",
             "deps": [
               {
                 "depType": "module",
                 "depName": "bitbucket.int.<company>.com/tf/s3",
                 "lookupName": "ssh://git@bitbucket.int.<company>.com/tf/s3.git",
                 "currentValue": "v1",
                 "datasource": "git-tags",
                 "depIndex": 0,
                 "updates": [],
                 "warnings": [
                   {
                     "topic": "bitbucket.int.<company>.com/tf/s3",
                     "message": "Failed to look up dependency bitbucket.int.<company>.com/tf/s3"
                   }
                 ],
                 "versioning": "hashicorp"
               },
               {"skipReason": "invalid-value", "depIndex": 1, "updates": []}
           ]
         }
``` 
This is the part of the main.tf it's getting the dependency from:
  ```hcl
module "ally_s3_bucket" {
source    = "git::ssh://git@bitbucket.int.<company>.com/tf/s3.git?ref=v1"
name      = "my-example-bucket"
namespace = "namespace-examples"
tags      = module.namespace.tags
}
```
Is it possible to somehow change the lookup name/template for managers such as terraform so that it just goes through https instead of ssh, or is that only reserved for regex? In that case would I be better just using the regex manager to accomplish what I'm trying to do, or am I not aware of some functionality that would come in handy here?

Thanks

**Comment by viceice** [comment, marked as answer]
Renovate can't use ssh based git lookups, so you have two options:

1. Disable terraform manager for that module and use regex manager
2. Use https url in terraform source files.
